docs/app/Examples/modules/Dropdown/Types/index.js

```diff
@@ -0,0 +1,100 @@
+    <ComponentExample
+      title='Selection'
+      description='A dropdown can be used to select between choices in a form.'
+      examplePath='modules/Dropdown/Types/DropdownExampleSelection'
+    />
+    <ComponentExample
+      title='Search Selection'
+      description='A selection dropdown can allow a user to search through a large list of choices.'
+      examplePath='modules/Dropdown/Types/DropdownExampleSearchSelection'
+    />
+    <ComponentExample
+      examplePath='modules/Dropdown/Types/DropdownExampleSearchSelectionTwo'
+    />
+    {/* Possibly add state selection example. */}
+    <ComponentExample
+      title='Multiple Selection'
+      description='A selection dropdown can allow multiple selections.'
+      examplePath='modules/Dropdown/Types/DropdownExampleMultipleSelection'
+    />
+    <ComponentExample
+      title='Multiple Search Selection'
+      description='A selection dropdown can allow multiple search selections.'
+      examplePath='modules/Dropdown/Types/DropdownExampleMultipleSearchSelection'
+    />
+    <ComponentExample
+      description='Dropdowns can support content that may not be allowed inside option tags.'
+      examplePath='modules/Dropdown/Types/DropdownExampleMultipleSearchSelectionTwo'
+    />
+    <ComponentExample
+      title='Search Dropdown'
+      description='A dropdown can be searchable.'
+      examplePath='modules/Dropdown/Types/DropdownExampleSearchDropdown'
+    />
+    <ComponentExample
+      title='Search In-Menu'
+      description='A dropdown can include a search prompt inside its menu.'
+      examplePath='modules/Dropdown/Types/DropdownExampleSearchInMenu'
+    >
+      <Message warning>
+        The example below shows the desired markup but is not functional.
+        Needs to be defined via shorthand, which is not yet possible.
+      </Message>
+    </ComponentExample>
+    <ComponentExample
+      description='A dropdown with multiple selections can include a search prompt inside its menu.'
+      examplePath='modules/Dropdown/Types/DropdownExampleMultipleSearchInMenu'
+    >
+      <Message warning>
+        The example below shows the desired markup but is not functional.
+        Needs to be defined via shorthand, which is not yet possible.
+      </Message>
+    </ComponentExample>
+    <ComponentExample
+      title='Inline'
+      description='A dropdown can be formatted to appear inline in other content.'
+      examplePath='modules/Dropdown/Types/DropdownExampleInline'
+    />
+    <ComponentExample
+      examplePath='modules/Dropdown/Types/DropdownExampleInlineTwo'
+    />
+    <ComponentExample
+      title='Pointing'
+      description='A dropdown can be formatted so that its menu is pointing.'
+      examplePath='modules/Dropdown/Types/DropdownExamplePointing'
+    >
+      <Message warning>
+        The example below shows (roughly) the desired markup but is not functional
+        since we don't currently support nested dropdowns.
+      </Message>
```

Comment:
Would be great to include some prompts to contribute in places like this.  A link to a PR search and the CONTRIBUTING.md would suffice.  You can reference [this Message in `ComponentExamples.js`](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Components/ComponentDoc/ComponentExamples.js#L25):

![image](https://cloud.githubusercontent.com/assets/5067638/19535336/0f84a092-95fc-11e6-80b7-94e9bdaf82a3.png)